atomic_reactor/plugins/pre_add_yum_repo_by_url.py

```diff
@@ -52,11 +52,25 @@ def run(self):
         if self.repourls:
             for repourl in self.repourls:
                 yumrepo = YumRepo(repourl)
-                yumrepo.fetch()
-                self.log.info("fetched repo from '%s'", yumrepo.repourl)
+                self.log.info("fetching yum repo from '%s'", yumrepo.repourl)
+                try:
+                    yumrepo.fetch()
+                except Exception as e:
+                    self.log.exception(
```

Comment:
Please put stuff that can be on a single line as a single line.

Also, consider
```
msg = "Failed to fetch yum repo {repo}: {exc}".format(repo=yumrepo.repourl, exc=e))
self.log.exception(msg)
raise RuntimeError(msg)
```

Also, is is really necessary to log an exception?

Reply:
I followed PEP8 (line < 80), but I can use rule line < 100 instead.

I have no strong opinion about logging exception there, I will remove it.

Reply:
Yeah, we use 100 in our projects.

Reply:
Updated